Tools/jit/template.c

```diff
@@ -80,6 +80,9 @@ do {                                                         \
 #undef JUMP_TO_ERROR
 #define JUMP_TO_ERROR() PATCH_JUMP(_JIT_ERROR_TARGET)
 
+#undef WITHIN_STACK_BOUNDS
+#define WITHIN_STACK_BOUNDS() 1
```

Comment:
The failing CI just didn't like the reference to `entry_frame`. I think this should work?

```suggestion
#define WITHIN_STACK_BOUNDS() (STACK_LEVEL() >= 0 && STACK_LEVEL() <= STACK_SIZE())
```

Reply:
We don't need to check this in JIT builds, so defining it to 1 is fine.

Reply:
Why don’t we need to check it in JIT builds?